src/test/skript/tests/syntaxes/expressions/ExprMaxStack.sk

```diff
@@ -2,3 +2,49 @@ test "max stack":
 	assert max stack size of diamond sword is 1 with "diamond sword max stack size failed"
 	assert max stack size of bucket is 16 with "bucket max stack size failed"
 	assert max stack size of dirt is 64 with "dirt max stack size failed"
+
+test "max stack override - itemtype" when running minecraft "1.20.5":
```

Comment:
I'd also like to see some tests of changing multiple items or category item types (all logs) at a time, with varying default sizes. Plus edge cases tests of null values on both sides.